tests/components/iaqualink/conftest.py

```diff
@@ -55,8 +56,8 @@ def get_aqualink_device(system, cls=None, data=None):
     if data is None:
         data = {}
 
-    num = random.randint(0, 999)
-    data["name"] = f"name_{num:03}"
+    name = "".join(random.choice(string.ascii_lowercase) for _ in range(8))
```

Comment:
There's also `homeassistant.util.uuid.random_uuid_hex()` if you'd rather have something really simple. (Assuming that would produce a valid identifier.)

Reply:
Could it not be a static id for the tests? Does it need to be random?
It suggest a logic issue in the tests

Reply:
@epenet Yes, it could be static. I could just pass the name as a parameter and make sure the caller doesn't use the same name twice.

The helper function @jkeljo mentioned should also work.

Reply:
We let make it static.

Reply:
Any progress with this?
I bumped into it three times already, latest in https://github.com/home-assistant/core/runs/4773324774?check_suite_focus=true